app/src/main/java/com/dessalines/thumbkey/utils/KeyboardLayout.kt

```diff
@@ -10,6 +10,7 @@ import com.dessalines.thumbkey.keyboards.KB_DE_MESSAGEEASE_SYMBOLS
 import com.dessalines.thumbkey.keyboards.KB_DE_THUMBKEY
 import com.dessalines.thumbkey.keyboards.KB_DE_THUMBKEY_MULTILINGUAL
 import com.dessalines.thumbkey.keyboards.KB_DE_TYPESPLIT
+import com.dessalines.thumbkey.keyboards.KB_ENDA_THUMBKEY_SYMBOLS
```

Comment:
This one is named different from the others.

Reply:
If you meant the missing underscore, I've just added it :+1: